src/i18n.js

```diff
@@ -54,6 +54,7 @@ i18n
           es: '0.7',
```

Comment:
I'd say we are ready to bump all to v1 😄 That way all of them are in sync again and we can go from version 1 onwards.

Reply:
done!